Simplification classes avions

martinregner opened the pull request:
Pour limiter les hypothèses et calculs (ex. moyennes par distance) peu lisible, choix d'une seule taille d'avion par distance conformément aux préconisations de Pierre.
Le nombre d'avions et de distances sont ainsi réduits sur la base de ces préco. et de la nouvelle version de la Base Carbone.

martinregner committed:
Simplification classes avions

Pour limiter les hypothèses et calculs (ex. moyennes par distance) peu lisible, choix d'une seule taille d'avion par distance conformément aux préconisations de Pierre.
Le nombre d'avions et de distances sont ainsi réduits sur la base de ces préco. et de la nouvelle version de la Base Carbone.

laem committed:
Correction du YAML

laem committed:
Correction des intervalles de distance

laem committed:
:art: Tentative de viz des trainées

laem committed:
Tentative d'explication du dégradé

laem commented:
Peut-être tester graphique un dégradé qui ne finit pas en blanc mais en mi-blanc mo-mauve ? Ce sera visuellement plus sympa, et après tout, la valeur de réchauffement additionnel retenue est une moyenne, ce serait donc un facteur de 3.7 qui devrait être considéré comme blanc (très peu probable).

martinregner committed:
Update ges-transport.yaml

laem committed:
Implémentation de l'incertitude de l'avion

Avec les vrai données.

laem committed:
Choix de la couleur de l'intercitude de l'avion

Voir l'explication dans le commentaire du choix, le blanc n'était pas
adapté du tout.

laem committed:
Affichage des notes en markdown

laem committed:
MAJ des paquets React refresh

laem committed:
Nouvelle version

Majeure, car on a cassé le schéma des données de l'avion
